esp/src/eclwatch/ClusterProcessesQueryWidget.js

```diff
@@ -0,0 +1,333 @@
+                store: ESPPreflight.CreateClusterProcessStore(),
+                columns: {
+                    col1: selector({
+                        width: 20,
+                        selectorType: 'checkbox'
+                    }),
+                    Configuration: {
+                        label: this.i18n.Configuration,
+                        renderHeaderCell: function (node) {
+                            node.innerHTML = Utility.getImageHTML("configuration.png", context.i18n.Configuration);
+                        },
+                        width: 10,
+                        sortable: false,
+                        formatter: function (clusterType) {
+                            if (clusterType === true) {
+                                return "<a href='#' />" + Utility.getImageHTML("configuration.png", context.i18n.Configuration) + "</a>";
+                            }
+                            return "";
+                        }
+                    },
+                    Name: tree({
+                        formatter: function (_name, row) {
+                            var img = "";
+                            var name = _name;
+                            if (row.type === "clusterProcess") {
+                                img = Utility.getImageHTML("server.png");
+                                name = row.Type + " - " + _name;
+                            } if (row.type === "machine") {
+                                img = Utility.getImageHTML("machine.png");
+                                name = "<a href='#' class='dgrid-row-url'>" + row.Netaddress + " - " + _name + "</a>";
+                            }
+                            return img + "&nbsp;" + name;
+                        },
+                        collapseOnRefresh: false,
+                        label: this.i18n.Name,
+                        sortable: true,
+                        width: 150
+                    }),
+                    Domain: {
+                        label: this.i18n.Domain,
+                        sortable: false,
+                        width: 100
+                    },
+                    Platform: {
+                        label: this.i18n.Platform,
+                        sortable: false,
+                        width: 75
+                    },
+                    ProcessNumber: {
+                        label: this.i18n.SlaveNumber,
+                        sortable: false,
+                        width: 100
+                    },
+                    Directory: {
+                        label: this.i18n.Directory,
+                        sortable: false,
+                        width: 200
+                    },
+                    LogDirectory: {
+                        label: this.i18n.LogDirectory,
+                        sortable: false,
+                        width: 200
+                    }
+                }
```

Comment:
Should there be one column without width?

Reply:
Removing a column width causes issues within the grid.